ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/Projects.resource

```diff
@@ -26,6 +26,8 @@ Open Data Science Projects Home Page
     Wait Until Page Contains    View your existing projects or create new projects.    timeout=30
     Maybe Wait For Dashboard Loading Spinner Page
 
+
```

Comment:
It should be only one line between keywords

Reply:
Resolved